projects/rawkode-academy/image-service/package.json

```diff
@@ -3,18 +3,18 @@
   "type": "module",
   "version": "0.0.1",
   "scripts": {
-    "prebuild": "cp node_modules/@resvg/resvg-wasm/index_bg.wasm src/wasm/resvg.wasm && cp node_modules/yoga-wasm-web/dist/yoga.wasm src/wasm/yoga.wasm",
-    "prestart": "cp node_modules/@resvg/resvg-wasm/index_bg.wasm src/wasm/resvg.wasm && cp node_modules/yoga-wasm-web/dist/yoga.wasm src/wasm/yoga.wasm",
+    "prebuild": "./scripts/copy_wasm.sh",
+    "prestart": "./scripts/copy_wasm.sh",
```

Comment:
Maybe instead of a scripts directory we should use a Justfile?

Reply:
Justfile wouldn't work. I want to run this step every time a `start` or `build` happens and Justfile doesn't work on a Cloudflare build, right?

Reply:
Yeah. `"prebuild": "just copy-wasm"` would work.

Reply:
I have no real objection, so here we go :)